### PR TITLE
Form Validation

### PR DESCRIPTION
**Pull request by Bhekisile**
Add html and style for the validation
Add validation function with check-points to script.js file
Create an error message

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -11,4 +11,142 @@ const getHam = document.querySelector('.getHam');
...
+  const email = document.form.email.value;
+  const comment = document.form.comment.value;
+  const errorMessages = [];
+
+  if (name.trim() === '') {
+    errorMessages.push('Name is required');
+  } else if (email.trim() === '') {
+    errorMessages.push('Email is required');
+  } else if (email !== email.toLowerCase()) {
+    errorMessages.push('Email must be in lowercase');
```
Comment (tufoinnkuo10) [inline script.js:141]: - [optional] you can make the uppercase warning to appear in red color so it can easily be spotted. nice job

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Team

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- error message ok
- form submits well
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — COMMENTED** [review]
Hi Chawki, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that appears already approved. Else, if reapproval is needed kindly leave a notice in the Pull Request or in the dashboard during submission.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AlphaNtihinduka — COMMENTED** [review]
Hi @chawkimbarki ,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
# Invalid Code Review Request ❌ 

- [ ] However much this is a pair programming activity, the submission should be individually. This means that @Bhekisile has to contribute in your repository and then you will need to submit your repository individually and not hers. Below is the screenshot for the project requirements. Kindly proof read them and incase of any question, you can reach out to me.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by Chawki-Mbarki** [comment]
Hey 👋@AlphaNtihinduka 
I think you didn't understand the problem.
The changes should be made in one repository only.
But it didn't get approved for me.
I don't have to implement those requirements in both repositories
They should only be applied in one repository.

Please read the submit section in project requirements.

**Review by AbelG101 — COMMENTED** [review]
# Hi STUDENT_MENTION 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- You have submitted a project, that should be implemented in both repositories (your partner's & yours) as specified in the below project submission mode so please make sure to implement the validation in your repository as well with your coding partners' commit.
    image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**